src/dlgTriggerEditor.cpp

```diff
@@ -43,19 +45,17 @@
 #include "dlgProfilePreferences.h"
 #include "dlgScriptsMainArea.h"
 #include "dlgTriggerPatternEdit.h"
-#include "SingleLineTextEdit.h"
-#include "TrailingWhitespaceMarker.h"
-#include "mudlet.h"
 #include "edbee/models/textdocumentscopes.h"
+#include "mudlet.h"
 
 #include "pre_guard.h"
+#include <QToolBar>
```

Comment:
If you are sorting alphabetically then this particular move doesn't make sense... :confused: